src/main/webapp/3Dscript.js

```diff
@@ -208,3 +191,33 @@ function animate() {
   renderer.render(scene, camera);
 };
 
+/**
+ * This function fetchs pose data from the RunInfo servlet,
+ * it is an asynchronous call requiring addMap() to be
+ * called after the data is fully loaded.
+ */
+function fetchData() {
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const id = urlParams.get('id');
+  const type = urlParams.get('dataType');
+  // Only need to refetch the data if it is not contained in local storage, in
+  // general it should be.
+  const isStored = urlParams.get('stored');
+  const subSectionNumber = urlParams.get('subSection');
+  console.log(id + type);
+  if (!isStored) {
```

Comment:
Prefer to test the positive condition first:
```
if (isStored) {
  ...
} else {
  ...
}
```
That way the reader doesn't have to do double negation.

https://drive.google.com/file/d/1-g637f-xAyGWbcILf5QbLRiTmvsDA3P9/view?usp=sharing